frontend/__tests__/api/authors.test.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { useMutation, useQuery } from "@apollo/client";
+import { render, screen } from "../../test-utils";
+import { useGetAuthors, useGetAuthor, useAddAuthor } from "../../api/authors";
+
+jest.mock("@apollo/client");
+
+describe("Authors test", () => {
+  const AuthorsTest = () => {
+    const { authors } = useGetAuthors();
+    const { author } = useGetAuthor();
+    const { addAuthor } = useAddAuthor();
+    addAuthor();
+    return authors && author ? <div>test</div> : null;
+  };
+  it("should use authors hook without error", () => {
+    const data = {};
+    useQuery.mockReturnValue({
+      loading: false,
+      error: null,
+      data,
+      refetch: () => {},
+    });
+    useMutation.mockReturnValue([
+      () => ({ data }),
+      { loading: false, error: null, data },
+    ]);
+    render(<AuthorsTest />);
+
+    expect(screen.getByText("test")).toBeInTheDocument();
+  });
+});
```

Comment:
Please re-examine your testing approach when testing things that don't necessarily have UI to render. Also please take a look at how you are using mocking. (line 6)